datafusion/expr/src/test/function_stub.rs

```diff
@@ -488,8 +490,61 @@ impl AggregateUDFImpl for Avg {
         &self.signature
     }
 
+    fn coerce_types(&self, arg_types: &[DataType]) -> Result<Vec<DataType>> {
```

Comment:
Technically this is a duplication with the avg logic, but I'm not sure why we need these stubs in the first place 🤔